packages/amplify-provider-awscloudformation/src/utils/remove-dependent-function.ts

```diff
@@ -0,0 +1,68 @@
+import { $TSAny, $TSContext, $TSObject, pathManager } from 'amplify-cli-core';
+import path from 'path';
+import * as TransformPackage from 'graphql-transformer-core';
```

Comment:
this namespace is a bit confusing. I would recommend destructuring just the functions you want or namespacing to `gqlTransformer` or something